packages/bot-skeleton/src/scratch/dbot-store.js

```diff
@@ -0,0 +1,40 @@
+class DBotStoreInterface {
```

Comment:
@Mahboobeh-binary can you clarify why both of these function couldn't be pass in from `quick_strategy_store` & `saveload_modal_store`?

Reply:
@sam-binary its been passed from those store but in this store we will have an default implementation for any things we required, so in case some one forget to pass it in we will se errors.

Reply:
@reza-fs that can be improved. in next PRs